app/HMS/Entities/User.php

```diff
@@ -70,6 +70,11 @@ class User implements AuthenticatableContract, CanResetPasswordContract, HasRole
     protected $account;
 
     /**
+     * @var \Doctrine\Common\Collections\ArrayCollection|Email[]
```

Comment:
SHould type this as \Doctrine\Common\Collections\Collection. The interface ArrayCollection implements. This'll help testing since Doctrine sometimes decides it needs to return a PersistantCollection (I think) which causes tests to blow up.